src/Twig/Runtime/TextRuntime.php

```diff
@@ -40,7 +41,7 @@ public function __construct(LoggerInterface $systemLogger, Slugify $slugify)
      */
     public function jsonDecode($string)
     {
-        return json_decode($string, true);
+        return Json::parse($string);
```

Comment:
Is throwing an exception ok here? Maybe people are using false returns here too...